crates/bazed-core/src/user_buffer_op.rs

```diff
@@ -48,4 +48,6 @@ pub(crate) enum Motion {
     BottomOfViewport,
     NextWordBoundary(WordBoundaryType),
     PrevWordBoundary(WordBoundaryType),
+    FindNext(hotsauce::Regex),
```

Comment:
These should take &Regex (&RegexCursor?) to allow stuff like caches.

Would also allow Copy again.